app/views/steps/shared/_saving_type_form.html.erb

```diff
@@ -0,0 +1,30 @@
+<% if show_none_option %>
```

Comment:
I think it would be better to duplicate the form in the each edit view, rather than create a partial with lots of conditionals.

Reply:
Thanks tim i've updated